cmake/Allowlist.cmake

```diff
@@ -0,0 +1,32 @@
+
+if(__caffe2_allowlist_included)
+  return()
+endif()
+
+set(__caffe2_allowlist_included TRUE)
+
+set(CAFFE2_ALLOWLISTED_FILES)
+if(NOT CAFFE2_ALLOWLIST)
+  return()
+endif()
+
+# First read the allowlist file and break it by line.
+file(READ "${CAFFE2_ALLOWLIST}" allowlist_content)
+# Convert file contents into a CMake list
+string(REGEX REPLACE "\n" ";" allowlist_content ${allowlist_content})
+
+foreach(item ${allowlist_content})
+  file(GLOB_RECURSE tmp ${item})
+  set(CAFFE2_ALLOWLISTED_FILES ${CAFFE2_ALLOWLISTED_FILES} ${tmp})
```

Comment:
You probably need also modify here: https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L147, right?